cluster/migrations/000001_m3-initial.up.sql

```diff
@@ -102,6 +102,17 @@ create table provisioning.disks
 
 comment on column provisioning.disks.capacity is 'Capacity in bytes';
 
+--  Table to store sessions of a <tenant>.user
 
+create table provisioning.sessions
```

Comment:
could you a comment beside each table column briefly describing it?

Reply:
done, let me know if that helps